lib/transport/transport-aux-data.h

```diff
@@ -31,7 +31,7 @@ typedef struct _LogTransportAuxData
   GSockAddr *peer_addr;
   GSockAddr *local_addr;
   gint proto;
-  gchar data[1024];
+  gchar data[1536];
```

Comment:
Based on how aux data is stored, `proxied-tcp` can increase aux-data with the following amount:

```
PROXIED_SRCIP - max 64 bytes
PROXIED_DSTIP - max 64 bytes
PROXIED_SRCPORT - max 8 bytes
PROXIED_DSTPORT - max 8 bytes
PROXIED_IP_VERSION - max 8 bytes
```

Storage size: `name_len + 1 + value_len + 1 + 1`

```
13 + 1 + 64 + 1 + 1 +
13 + 1 + 64 + 1 + 1 +
15 + 1 + 8 + 1 + 1 +
15 + 1 + 8 + 1 + 1 +
18 + 1 + 8 + 1 + 1 =

241
```

👍🏻